tests/foreman/ui/test_adusergroup.py

```diff
@@ -54,17 +53,25 @@ def setUpClass(cls):  # noqa
 
     def tearDown(self):
         with Session(self.browser) as session:
-            session.nav.go_to_user_groups()
-            if self.usergroup.search(self.usergroup_name):
-                self.usergroup.delete(self.usergroup_name, True)
-            if self.usergroup.search(self.usergroup_name2):
-                self.usergroup.delete(self.usergroup_name2, True)
             set_context(session, org=ANY_CONTEXT['org'])
-            session.nav.go_to_users()
             if self.user.search(self.ldap_user_name):
                 self.user.delete(self.ldap_user_name)
+            if self.usergroup.search(self.usergroup_name):
+                self.usergroup.delete(self.usergroup_name, True)
         super(ActiveDirectoryUserGroupTestCase, self).tearDown()
 
+    def check_external_user(self):
```

Comment:
More as validation rather than thing that do real actions. If external user is unavailable due any reason then it doesn't make sense to run test case at all. It is not possible to move it to setup area as browser entity is accessible only after whole chain of setups is executed. +Only after first login, user become visible in the system. There are some algorithm that can activate it using API and CLI, but they are still not tested at full capacity and doesn't guarantee same behavior that we have through UI. Anyway, we are working here with UI, so it is more appropriate to follow proposed approach